velox/exec/PartitionedOutputBufferManager.cpp

```diff
@@ -146,6 +155,24 @@ void releaseAfterAcknowledge(
 }
 } // namespace
 
+PartitionedOutputBuffer::PartitionedOutputBuffer(
+    std::shared_ptr<Task> task,
+    bool broadcast,
+    int numDestinations,
+    uint32_t numDrivers)
+    : task_(std::move(task)),
+      broadcast_(broadcast),
+      numDrivers_(numDrivers),
+      maxSize_(std::max<int64_t>(
+          FLAGS_min_partitioned_output_buffer_mb << 20,
```

Comment:
It is hard to keep track of various gflags. Let's not use these. 

Also, let's consolidate all the config logic inside config().maxPartitionedOutputBufferSize().